react/features/toolbox/functions.web.js

```diff
@@ -138,6 +138,19 @@ export function getToolbarClassNames(props: Object) {
 }
 
 /**
+ * Indicates if a toolbar button is enabled.
+ *
+ * @param {string} name - The name of the setting section as defined in
+ * interface_config.js.
+ * @returns {boolean} - True to indicate that the given toolbar button
+ * is enabled, false - otherwise.
+ */
+export function isButtonEnabled(name) {
```

Comment:
What did this change of moving the function provide you? I'm not against it but I also don't understand why you moved it.